test/unit/com/unifina/service/SignalPathServiceSpec.groovy

```diff
@@ -29,12 +29,12 @@ class SignalPathServiceSpec extends Specification {
 	CanvasService canvasService
 
 	def setup() {
-		me = new SecUser(username: "me@streamr.com", password: "pw", name: "name")
+		me = new SecUser(username: "me@streamr.com", password: "pw", name: "name", lastLogin: new Date())
```

Comment:
Can remove `lastLogin` if you set the default field value in `SecUser.groovy` (also elsewhere in this file)